src/routes/zome_call.rs

```diff
@@ -91,11 +91,11 @@ pub async fn zome_call(
     } = params;
     // Check payload byte length does not exceed configured maximum.
     if let Some(payload) = &query.payload {
-        if payload.as_bytes().len() > state.configuration.payload_limit_bytes as usize {
-            return Err(HcHttpGatewayError::PayloadSizeLimitError {
-                size: payload.as_bytes().len() as u32,
-                limit: state.configuration.payload_limit_bytes,
-            });
+        if payload.len() > state.configuration.payload_limit_bytes as usize {
```

Comment:
Hmm, this is correct I think but it's not that clear why. Because we're accepting base64, we know there are no multi-byte UTF characters. Could you maybe leave a comment here that the string length is the same as the byte length because we know the format of the string?

Reply:
Added comments and fixed the char calc https://github.com/holochain/hc-http-gw/pull/25/commits/d06199f68a6e3ceaa7242d246b1ca36c0ae1b439